cmake/partition_manager.cmake

```diff
@@ -263,9 +263,22 @@ foreach(part ${PM_ALL_BY_SIZE})
     list(APPEND explicitly_assigned ${part})
   else()
     if(${part} IN_LIST images)
-      set(${part}_PM_HEX_FILE ${${part}_ZEPHYR_BINARY_DIR}/${${part}_KERNEL_HEX_NAME})
+      set(${part}_PM_HEX_FILE ${PROJECT_BINARY_DIR}/${part}.hex)
       set(${part}_PM_ELF_FILE ${${part}_ZEPHYR_BINARY_DIR}/${${part}_KERNEL_ELF_NAME})
-      set(${part}_PM_TARGET ${part}_subimage)
+      set(${part}_PM_TARGET ${part}_hex)
+      add_custom_command(
```

Comment:
Why is this needed ?
Also, please add some more description in the commit message.

I don't see how this commit relates to: NCSDK-5567
can you elaborate a bit.

Reply:
It was my understanding from reading the issue that the intended behavior was that a copied hex file should be created, and that it was missing. I don't really have much justification apart from that, so I am happy to drop the change if it is not useful.

Reply:
@oyvindronningstad can you join here ?
what would be the reason for copying the file.